kubernetes/resource_kubernetes_deployment_test.go

```diff
@@ -2715,11 +2714,27 @@ func testAccKubernetesDeploymentConfig_beforeUpdate(name, imageName string) stri
 
           resources {
             limits {
-              memory = "512M"
+              memory = "128Mi"
```

Comment:
I changed this just because the test was failing sometimes on my local machine because the kind cluster was running short on memory.